Source/Scene/GlobeSurfaceTile.js

```diff
@@ -245,6 +245,13 @@ define([
         var surfaceTile = tile.data;
         if (!defined(surfaceTile)) {
             surfaceTile = tile.data = new GlobeSurfaceTile();
+            surfaceTile.tileBoundingBox = new TileBoundingBox({
+                rectangle : tile.rectangle,
+                ellipsoid : tile.tilingScheme.ellipsoid
+            });
+            // Set the distance of the tile now in order to prioritize the request later.
```

Comment:
Can we also use the min/max height of the parent tile?  I'm afraid this is going to be way off, for example, when zoomed in close on Mount Everest.

@kring any thoughts here?